FinSearchUnified/Tests/Helper/Utility.php

```diff
@@ -60,4 +60,15 @@ public static function sResetArticles()
         } catch (\Exception $ignored) {
         }
     }
+
+    public static function assertMinimumVersion($requiredVersion)
```

Comment:
I think this can be removed since it is not used in `QueryBuilderFactory` anymore.